lib/logmsg/tests/test_tags.c

```diff
@@ -159,55 +141,66 @@ test_msg_tags(void)
 
 }
 
-void
-test_filters(gboolean not)
+ParameterizedTestParameters(tags, test_filters)
+{
+  const gboolean not[] =
```

Comment:
This array must be `static` otherwise if you have ASLR enabled it is going to fail.

Reply:
Okay. I will change it.